src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -62,6 +62,14 @@ public FuzzyQueryBuilder(String name, Object value) {
     }
 
     /**
+     * package private constructor for serialization only
+     */
+    FuzzyQueryBuilder() {
+        this.name = null;
+        this.value = null;
```

Comment:
I think I like the idea of moving the INSTANCE (i'd rather call them PROTOTYPE) constants from the parsers to the builders, then use existing constructors. However, then we will have to make them public (or at least package private) to access them from the parsers. If your think this is a good idea I'll go through all the classes again and change this.